wire-runtime/src/main/java/com/squareup/wire/Message.java

```diff
@@ -142,8 +142,8 @@ public ByteString buildUnknownFields() {
   }
 
   /** <b>For generated code only.</b> Utility method to return a mutable copy of {@code list}. */
-  protected static <T> List<T> copyOf(List<T> list) {
-    if (list == null) throw new NullPointerException("list == null");
+  protected static <T> List<T> copyOf(String name, List<T> list) {
+    if (list == null) throw new NullPointerException(name + " == null");
```

Comment:
This one really isn't needed, but I guess it doesn't hurt, and it makes explicit the contract.